skll/learner/__init__.py

```diff
@@ -67,6 +67,18 @@
 from skll.data.dict_vectorizer import DictVectorizer
 from skll.data.readers import safe_float
 from skll.metrics import _SCORERS
+
+# import types when type checking but not otherwise to avoid circular imports
+# if TYPE_CHECKING:
```

Comment:
this should be commented out?

Reply:
Nope, we don't need it. I will remove it.